src/test/java/Search.java

```diff
@@ -0,0 +1,105 @@
+import InternSearch.pages.LoginPageSearch;
+import InternSearch.pages.SearchPageSearch;
+import org.openqa.selenium.By;
+import org.openqa.selenium.Keys;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+import java.util.ArrayList;
+import java.util.List;
+
+
+public class Search extends BaseTest {
+    String songName = "Dark Days";
+    //use all cases- exact spelling of song name, ingnore trailing and hading white space
+    String [] list = {"Dark Days"," Dark Days ","Dark Days "};
+
+    String [] caseSenList = {"darks day", "darKs DAy"};
+    @Test(enabled = true, priority = 0)// dataProvider = "songTitles", dataProviderClass = BaseTest.class)
+    public void searchSongTitle() throws InterruptedException{
```

Comment:
Please, delete `throws InterruptedException`

Reply:
Thanks Alex, I deleted it.